WooCommerce/UITestsFoundation/XCTest+Extensions.swift

```diff
@@ -153,4 +153,10 @@ extension XCUIElement {
         '\(firstSubstring)' and '\(secondSubstring)' does not appear on label!
         """)
     }
+
+    /// timeout: timeout value, if not specified defaults to 10
```

Comment:
I think, if you're going for the documentation style comment, it makes sense to give it a more formal look, and more consistent with the other exampled of documentation (inspired by `waitForExistence` from XCTest):

```suggestion
    /** Waits the specified amount of time (defaults to 10) for the element's isHittable property to be true, and then taps it. */
```

Reply:
yup that does look better! thanks for pointing to that, but i also want it to show the parameter value, i updated the comment in https://github.com/woocommerce/woocommerce-ios/pull/9621/commits/1d5d3f9a85f49e8de9059b47e16307590403d0f2 (only comment changes)